soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala

```diff
@@ -170,9 +170,9 @@ case object SodaLocation extends SodaType with ((Option[String], Option[String],
 
     def decode(v: JValue) = pointPattern.matches(v).orElse(Pattern.matches(v)).map { results =>
       val coords = for {
-        lat <- latitude.get(results)
         lon <- longitude.get(results)
-      } yield (lat, lon)
+        lat <- latitude.get(results)
+      } yield (lon, lat)
```

Comment:
..but when putting them into a sodalocation, they _should_ be lat/lon, at least as long as sodalocation's being used for both location and point (which is itself sorta weird, since they're really not the same, as a point dones't have any of the other fields of a point)

Reply:
..and indeed SodaLocation's json _encoder_ won't generate a valid point.  It really needs to be a separate type.